docs/core/toc.yml

```diff
@@ -238,16 +238,17 @@
   items:
   - name: Overview
     href: tools/index.md
-  - name: Tools
+  - name: Global and local tools
     expanded: true
     items:
-    - name: Global tools
-      expanded: true
-      items:
-      - name: Overview
-        href: tools/global-tools.md
-      - name: Create a global tool
-        href: tools/global-tools-how-to-create.md
+    - name: Overview
+      href: tools/global-tools.md
+    - name: Create a tool
```

Comment:
I would put the three tutorials under a "Tutorials" node. Otherwise the install/use headings will draw readers, but the basic info about installing is actually in the overview topic. BTW, maybe the install info should be broken out into a separate topic, since it's not usually in an overview topic in my experience.

Reply:
Good point. The overview isn't really an overview except for the first paragraph, it's really a how-to.  I'll change the name to fit the content and put the tutorials in a tutorials node.